### PR TITLE
24 dBM instead of 23

### PR DESCRIPTION
**Pull request by jlpoltrack**
As mentioned here: #57

### CODE REVIEW

File: mLRS/Common/hal/rx-hal-R9M-868-f103c8.h [hunk 1]
```diff
@@ -239,8 +239,10 @@ void rfpower_calc(int8_t power_dbm, uint8_t* sx_power, int8_t* actual_power_dbm,
     voltage_mV = 1730;
     *actual_power_dbm = 27;
   } else if (power_dbm > 22) {
-    voltage_mV = 1390;
-    *actual_power_dbm = 23;
+    //voltage_mV = 1390;
+    //*actual_power_dbm = 23;
+    voltage_mV = 1475;
```
Comment (olliw42) [inline mLRS/Common/hal/rx-hal-R9M-868-f103c8.h:244]: did you guesstimate this value, or did you had a chance to adjust it using a power meter?

File: mLRS/Common/hal/rx-hal-R9M-868-f103c8.h [hunk 2]
```diff
@@ -276,7 +278,7 @@ const rfpower_t rfpower_list[] = {
     { .dbm = POWER_3_DBM, .mW = 2 },
     { .dbm = POWER_10_DBM, .mW = 10 },
     { .dbm = POWER_20_DBM, .mW = 100 },
-    { .dbm = POWER_23_DBM, .mW = 200 },
+    { .dbm = POWER_24_DBM, .mW = 251 },
```
Comment (olliw42) [inline mLRS/Common/hal/rx-hal-R9M-868-f103c8.h:281]: I guess 250 should be accurate enough and potentially less confusing
dito in below occurences

File: mLRS/Common/hal/rx-hal-R9M-868-f103c8.h (hunk 1 above)
Comment (jlpoltrack) [reply mLRS/Common/hal/rx-hal-R9M-868-f103c8.h:244]: Guessed 1450, needed 1475 to be consistently over 24 dBm on the IRC power meter.  I now recall there were concerns on R9 overheating, can lower have a safer value if we want.

File: mLRS/Common/hal/rx-hal-R9M-868-f103c8.h (hunk 2 above)
Comment (jlpoltrack) [reply mLRS/Common/hal/rx-hal-R9M-868-f103c8.h:281]: Fair enough, will update.

**Comment by olliw42** [comment]
many thx, sir !!!

**Pull request merged**